package.json

```diff
@@ -49,17 +49,17 @@
     "es6-symbol": "^3.1.1",
     "escape-html": "^1.0.3",
     "express": "^4.15.2",
-    "extract-text-webpack-plugin": "^2.1.2",
+    "extract-text-webpack-plugin": "^3.0.0",
     "file-loader": "^0.11.2",
     "font-awesome": "^4.7.0",
     "glob": "^7.1.1",
     "http-link-header": "^0.8.0",
     "immutable": "^3.8.1",
     "intersection-observer": "^0.3.2",
     "intl": "^1.2.5",
-    "intl-relativeformat": "^1.3.0",
+    "intl-relativeformat": "^2.0.0",
```

Comment:
- [Release v2.0.0 — Upgraded to Latest intl-messageformat · yahoo/intl-relativeformat](https://github.com/yahoo/intl-relativeformat/releases/tag/v2.0.0)

> - Upgrade intl-messageformat to ^2.0.0 ([#50](https://github.com/yahoo/intl-relativeformat/pull/50), **@ jasonmit**)